README.md

```diff
@@ -87,7 +87,7 @@ machine.
 The unit of compilation and execution in Oak is an **Oak Module**. Each Oak
 Module is a self-contained
 [WebAssembly module](https://webassembly.org/docs/modules/) that is interpreted
-by an Oak VM instance as part of an Oak Node.
+by an Oak VM instance as part of an Oak Application.
```

Comment:
I am wondering if calling it a Oak VM might lead people to expect certain things. Is it actually a VM?

Reply:
Yeah I think we should just call it Oak Node or something like that and drop the term VM entirely.